ocs_ci/ocs/ui/base_ui.py

```
@@ -680,6 +681,53 @@ def refresh_page(self):
+        original_handles = set(driver.window_handles)
+
+        self.do_click(locator, enable_screenshot=True)
+
+        try:
+            WebDriverWait(driver, timeout).until(
+                lambda d: len(d.window_handles) > len(original_handles)
+            )
+        except TimeoutException:
+            raise TimeoutException(
+                f"No new tab opened within {timeout}s after clicking {locator}"
+            )
```

Ruff (0.16.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)